user-service/src/main/java/rs/edu/raf/banka1/model/MarketOrder.java

```diff
@@ -24,7 +30,12 @@ public class MarketOrder {
     private Double fee;
     private Double price;
     private Boolean allOrNone;
-    private Long lastModifiedDate;
+
+    @UpdateTimestamp
+    private Instant updatedAt;
```

Comment:
Instant je u UTC zoni? Jel ima razlog zasto niste isli sa unix timestamp? Do sad smo to koristili za vreme

Reply:
@Kovelja009 upravo.
@ijevtic mislim da sam negde procitao da @UpdateTimestamp radi sa Instantima pa sam ga zato upotrebio